draft-irtf-cfrg-opaque.md

```diff
@@ -624,45 +612,31 @@ Output:
 
 Steps:
 1. N = Unblind(blind, response.data)
-2. y = Finalize(pwdU, N, "OPAQUE00")
+2. y = Finalize(pwdU, N, "OPAQUE01")
 3. rwdU = HKDF-Extract("rwdU", Harden(y, params))
-4. Create secret_credentials with CredentialExtensions matching that
-   contained in response.secret_credentials_list
-5. Create cleartext_credentials with CredentialExtensions matching that
-   contained in response.cleartext_credentials_list
-6. pt = SerializeExtensions(secret_credentials)
+4. Create secret_credentials with credentials matching those needed to
+   construct the `SecretCredentials` structure
+5. Create cleartext_credentials with credentials matching those needed to
+   construct the `CleartextCredentials` structure
+6. pt = Serialize(secret_credentials)
 7. nonce = random(32)
 8. pseudorandom_pad = HKDF-Expand(rwdU, concat(nonce, "Pad"), len(pt))
 9. auth_key = HKDF-Expand(rwdU, concat(nonce, "AuthKey"), Nh)
 10. export_key = HKDF-Expand(rwdU, concat(nonce, "ExportKey"), Nh)
 11. ct = xor(pt, pseudorandom_pad)
-12. auth_data = SerializeExtensions(cleartext_credentials)
-13. Create InnerEnvelope contents with (nonce, ct, auth_data)
-14. t = HMAC(auth_key, contents)
+12. auth_data = Serialize(cleartext_credentials)
+13. Create InnerEnvelope contents with (nonce, ct)
```

Comment:
If I'm not mistaken, the InnerEnvelope should also specify the InnerEnvelopeMode, doesn't it?

Reply:
Good catch, thank you! Added to parameters.